docs/ht_ww.md

```diff
@@ -58,4 +102,23 @@ If enabling a wakeword, be sure to also set `"listen": true`.
 
 Multiple hotwords can be configured at the same time, even the same word with different plugins.  This allows for more accurate ones to be used before the less accurate, but only if the plugin is installed.
 
+
+### Example hotword config (keyboard wakeword)
```

Comment:
Should be ## instead of ###, as this is not a sub-heading of "List of OVOS WakeWord Plugins"